apps/prairielearn/src/ee/lib/validateHTML.ts

```diff
@@ -5,7 +5,11 @@ type ChildNode = parse5.DefaultTreeAdapterMap['childNode'];
 
 const mustacheTemplateRegex = /^\{\{.*\}\}$/;
 const mustacheTemplateExtractorRegex = /\{\{((?:[^}]|\}[^}])*)\}\}/g;
-const answersNameExtractorRegex = /answers-name="([^"]*)"/g;
+
+interface ValidationResult {
```

Comment:
Add a docstring to describe the properties here? Does `mandatoryPythonCorrectAnswers` need to be set even if there are errors?

Reply:
Done in 5cbebd2!